builder/Dockerfile.base

```diff
@@ -75,23 +70,10 @@ RUN mkdir /tmp/busybox \
 # PEP600 compatible wheels for our linux platform, or we start building orjson
 # from source using a rust toolchain.
 RUN pip3 install -U pip==20.2.4 pip-tools==5.3.1
-RUN npm config set user 0 && \
-    npm config set unsafe-perm true && \
-    npm install -g rollup@2.9.1 rollup-plugin-terser@5.3.0
 RUN curl --fail -L https://dl.google.com/go/go1.17.1.linux-amd64.tar.gz | tar -C /usr/local -xzf -
 
 RUN curl --fail -L https://storage.googleapis.com/kubernetes-release/release/v1.22.2/bin/linux/amd64/kubectl -o /usr/bin/kubectl && \
-    chmod a+x /usr/bin/kubectl && \
-    kubectl completion bash > /usr/share/bash-completion/completions/kubectl
-RUN curl --fail -L https://raw.githubusercontent.com/ahmetb/kubectx/v0.7.0/kubectx -o /usr/local/bin/kubectx && chmod a+x /usr/local/bin/kubectx && \
-    curl --fail -L https://raw.githubusercontent.com/ahmetb/kubectx/v0.7.0/completion/kubectx.bash -o /usr/share/bash-completion/completions/kubectx
-RUN curl --fail -L https://raw.githubusercontent.com/ahmetb/kubectx/v0.7.0/kubens -o /usr/local/bin/kubens && chmod a+x /usr/local/bin/kubens && \
-    curl --fail -L https://raw.githubusercontent.com/ahmetb/kubectx/v0.7.0/completion/kubens.bash -o /usr/share/bash-completion/completions/kubens
```

Comment:
Being able to run `kubectl` from the pod has been handy for me, FTR. I don't feel that strongly about it, but it's been handy.

Reply:
`kubectl` is still in the pod (but the bash-completions for it aren't).  It's `kubectx` and `kubens` that are going away.